examples/src/examples/graphics/physical-light-units.tsx

```diff
@@ -0,0 +1,303 @@
+import React from 'react';
+import * as pc from '../../../../';
+
+import { BindingTwoWay } from '@playcanvas/pcui';
+import { BooleanInput, LabelGroup, Panel, SliderInput } from '@playcanvas/pcui/react';
+import { Observer } from '@playcanvas/observer';
+class PhysicalLightUnitsExample {
+    static CATEGORY = 'Graphics';
+    static NAME = 'Physical Light Units';
```

Comment:
maybe call it 'Light physical' to show them next to 'Lights' example?

Reply:
Done.